packages/fetchai/connections/p2p_noise/connection.yaml

```diff
@@ -0,0 +1,27 @@
+name: p2p_noise
+author: fetchai
+version: 0.1.0
+description: The p2p noise connection implements an interface to standalone golang
+  noise node that can exchange aea envelopes with other agents participating in the
+  same p2p network.
+license: Apache-2.0
+aea_version: '>=0.3.0, <0.4.0'
+fingerprint:
+  __init__.py: QmbPzrjd27coFfS2vN9xDL4uARKZWCCmtWvmEuzGKSjxf7
+  aea/api.go: QmdatRSHu3Nuz18NDMk9PGSPi34o1ATggYRX962nP6jSXm
+  connection.py: QmPi2M1uaexKUGjZxpSuoDTZ6J6wVYYCpEREp9TZYY3jwu
+  noise_node.go: QmP92aiWBd2LNgEbH1gSjr3H575c4XUPkZ2uZVTq3QgUWJ
+  publickey.py: QmPyfGWQJAhHV1SQoNiCkS5uDhFXSJfPrPe7igQ1uxcPsa
+fingerprint_ignore_patterns: []
```

Comment:
shouldn't we ignore the log file? Or other files generated from the `.go` source code?

Reply:
The log file won't live in the package but rather in the agent root directory.
I am not aware of go source code generating other files.